bookmarks/tests/test_website_loader.py

```diff
@@ -94,3 +98,10 @@ def test_load_website_metadata_trims_title_and_description(self):
             metadata = website_loader.load_website_metadata('https://example.com')
             self.assertEqual('test title', metadata.title)
             self.assertEqual('test description', metadata.description)
+
+    def test_load_website_metadata_using_og_description(self):
```

Comment:
Ideally there would be a test that verifies that `name="description"` has precedence over the Open Graph metadata.

Reply:
Done